privaterelay/templates/includes/banners.html

```diff
@@ -1,3 +1,16 @@
+    {% if last_bounce_type %}
+<div class="dashboard-banners flx hide-750">
+  <div class="banner-gradient-bg">
+    <div class="bounced-email-banner banner-content flx flx-row">
+      <div class="banner-bounce-warning banner-logo"></div>
+      <div class="banner-copy flx flx-col">
+        <p class="banner-hl ff-Met">Relay couldn't deliver your email.</p>
+        <p class="banner-sub">On {{ last_bounce_date|date:"SHORT_DATETIME_FORMAT" }}, Relay received a <em>{{ last_bounce_type }}</em> "bounce" from your email provider when trying to deliver your email. This can happen if Relay couldn't connect to your email provider, or if your mailbox was full. Relay will try to resume delivering email to you at {{ next_email_try|date:"SHORT_DATETIME_FORMAT" }}</p>
+      </div>
```

Comment:
I find this copy confusing even though I know what you're trying to say. The main confusion is which way things aren't working. I would suggest something like:

```
We are currently unable to send email to your forwarding email address myrealaddress@gmail.com. We have received a <em>{{ last_bounce_type }}</em> "bounce" from your email provider @gmail.com when trying to forward relay emails to you. This can happen if we couldn't connect to your email provider, or your mailbox was full. We will try again on {{ next_email_try|date:"SHORT_DATETIME_FORMAT" }}.
```